src/components/Video/Video.module.scss

```diff
@@ -0,0 +1,71 @@
+.frame {
+  display: block;
+  position: relative;
+  contain: layout style paint;
+  overflow: hidden;
+  width: auto;
+  height: auto;
+  > div {
+    width: 100%;
+    height: 100%;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    object-fit: cover;
+  }
+
+  // overwrite playerjs styles to make the video fluid
+  :global(.video-js) {
+    width: 100%;
+    height: 100% !important;
+    padding: 0 !important;
+
+    video {
+      position: relative;
+    }
+
+    // Custom video player styling
+    :global(.vjs-control) {
+      height: 3em;
+    }
+
+    :global(.vjs-control-bar) {
+      height: 3.5em;
+      background-color: none;
+      background: linear-gradient(transparent, rgb(0 0 0 / 0.35));
+    }
+
+    :global(.vjs-control):focus {
+      text-shadow: none;
+    }
+  }
+}
+
+.brokenVideo {
+  width: 100%;
+  aspect-ratio: 16/9;
+  height: 100%;
+  background: lab(94.1 0 0);
```

Comment:
Can we not make this a CSS variable, copying it to the oldlayout class? Eventually we will merge the two layout classes and will make it easier if we're already using the colours everywhere.